### PR TITLE
[2.7.11] Disable helm chart publishing on tag and promote events

### PR DESCRIPTION
**Pull request by HarrisonWAffel**
## Problem
The current logic for publishing helm charts on tag events relies on various images being present in Dockerhub. Now that we are using a staging registry, this pipeline stage will fail. Similarly, because we are not publishing images to dockerhub, the `promote-docker-image` pipeline will also fail 
 
## Solution
Temporarily disable the helm chart publish pipeline stage. 

### Automated Testing

* Test types added/modified:
    * Unit
    * Integration (Go Framework)
    * Integration (v2prov Framework)
    * Validation (Go Framework)
    * Other - Explain: _EXPLAIN_
    * None
    * _REMOVE NOT APPLICABLE BULLET POINTS ABOVE_
* If "None" - Reason: _EXPLAIN THE REASON_
  
* If "None" - GH Issue/PR: _LINK TO GH ISSUE/PR TO ADD TESTS_

Summary: _TODO_

### Regressions Considerations

_TODO_

Existing / newly added automated tests that provide evidence there are no regressions:
* _TODO_

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1119,19 +1119,19 @@ steps:
...
-    - allUsers:READER
-    cache_control: "public,no-cache,proxy-revalidate"
-    source: bin/chart
-    target: releases.rancher.com/server-charts
-    token:
-      from_secret: google_auth_key
-  when:
-    event:
-    - tag
+#- name: chart-publish
```
Comment (briandowns) [inline .drone.yml:1122]: Can this be removed in favor of bringing it back from a previous version if necessary?

Comment (HarrisonWAffel) [reply .drone.yml:1122]: Sure, I'll completely remove it instead of commenting it out 👍

Comment (HarrisonWAffel) [reply .drone.yml]: Okay, I've removed the commented out portions

**Review by snasovich — APPROVED** [review]

File: .drone.yml [hunk 2]
```diff
@@ -1297,88 +1283,3 @@ trigger:
...
-- name: docker
-  host:
-    path: /var/run/docker.sock
-
-trigger:
-  event:
-  - promote
----
-kind: pipeline
-name: promote-docker-image
```
Comment (jiaqiluo) [inline .drone.yml:1348]: This pipeline is not related to charts, do we want to include it in this PR?  Or if it is requested, I suggest to update the commit message to make it clear.

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 2 above)
Comment (jiaqiluo) [reply .drone.yml:1348]: OK, this is requested. :)

**Review by jiaqiluo — APPROVED** [review]

File: .drone.yml (hunk 2 above)
Comment (HarrisonWAffel) [reply .drone.yml:1348]: Discussed in slack, it was agreed to remove all steps related to promoting charts and images, as they will likely fail
